data_service/src/schema.rs

```diff
@@ -1,5 +1,15 @@
 // @generated automatically by Diesel CLI.
 
+diesel::table! {
+    champion_matchup_data (champion_name) {
+        champion_name -> Varchar,
+        opposing_champion_name -> Varchar,
+        gold_difference -> Int4,
```

Comment:
Should adjust table to have champion gold as a column instead of gold diff

Reply:
i will assign the ticket to you